src/app/connections/connections.component.ts

```diff
@@ -31,11 +32,21 @@ export class ConnectionsComponent implements OnInit {
     public context: ContextService,
     private http: HttpClient,
     private delegatedAuthenticationService: DelegatedAuthenticationService,
-    private alertService: AlertService
+    private alertService: AlertService,
+    private route: ActivatedRoute
   ) {
   }
 
   ngOnInit(): void {
+      this.route.queryParams.subscribe(params => {
+        if (params.destinationToken) {
```

Comment:
Add null/empty check for params.silentRefreshRedirectUri, params.destination, params.destinationToken